org.eclipse.scanning.api/src/org/eclipse/scanning/api/points/models/AbstractPointsModel.java

```diff
@@ -67,7 +68,7 @@ public String getSummary() {
 		buf.append(names);
 		return buf.toString();
 	}
-	
+
 	@Override
 	public int hashCode() {
 		final int prime = 31;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)